### ISSUE TITLE
New Client Website Project: What to include in a discovery questionnaire

### ISSUE THREAD

**Issue by liamdempsey** [burbswp/main-presentations#15]
It would be really interesting to learn what other companies and freelancers are using as part of their new project discovery process. What sort of questions or conversations are you holding after a project has been approved? More specifically, how are you getting at discerning brand identity, key messages, target audiences, etc. If you have a questionnaire, what does that look like? Is it an online form? Something offline? Any sort of insight along this line of thinking would be really interesting to me.

**Comment by MattRyanCo** [comment]
I like this idea. My questionnaire has done well by me and helps to weed out the tire kickers from the serious inquiries.

**Comment by AmyEtcetera** [comment]
I think this would be a great panel discussion.

**Comment by liamdempsey** [comment]
@MattRy: I'm interested in learning what sort of questionnaire you have for clients AFTER you've signed them up. What sort of written document do you use as part of your, for example, new website project discovery?

**Comment by MattRyanCo** [comment]
uh...new website project discovery? Do you mean credentials for access to services, or something else?

**Comment by liamdempsey** [comment]
No, Matt. I am talking more about getting to how the client envisions success, how they define their target audience, what they consider to be their key marketing messages and how they see themselves from a marketing perspective.

Does that help to clarify my aims?

**Comment by MattRyanCo** [comment]
@liamdempsey Got it. Yep.. My questionnaire does go into these areas. I ask for the info up front, but typically I need to go into it much deeper during a project kick-off conversation. Here's my form: 
http://capwebsolutions.com/website-project-profile/
